test_battleship.py

```diff
@@ -71,7 +72,8 @@ def test_read_value_without_config_file(self):
 
 # validate
 
-    def test_validate_game_difficulty(self):
+    @patch('BattleShip.BattleShip.validate_carrier_points', return_value=True)
```

Comment:
@skelkelian do you plan on adding other mocking calls in a future PR?

Reply:
Yes I will be adding other mocking calls in a future PR!